products/nodejs.md

```diff
@@ -17,6 +17,12 @@ command: node --version
 releaseDateColumn: true
 sortReleasesBy: 'releaseCycle'
 releases:
+  - releaseCycle: "18"
+    release: 2022-04-19
+    lts: true
```

Comment:
could you make this false and add a comment blow to toggle it to lts after 2022-10-25 ?

Reply:
Done